spec/javascripts/services/short_form_application_service_spec.coffee

```
@@ -37,19 +37,19 @@ do ->
       listing:
         Id: ''
       loadListing: ->
-      preferenceMap = {
-        'certOfPreference': 'Certificate of Preference (COP)',
-        'displaced': 'Displaced Tenant Housing Preference (DTHP)',
-        'liveWorkInSf': 'Live or Work in San Francisco Preference',
-        'liveInSf': 'Live or Work in San Francisco Preference',
-        'workInSf': 'Live or Work in San Francisco Preference',
-        'neighborhoodResidence': 'Neighborhood Resident Housing Preference (NRHP)',
-        'assistedHousing': 'Rent Burdened / Assisted Housing Preference',
-        'rentBurden': 'Rent Burdened / Assisted Housing Preference',
-        'antiDisplacement': 'Anti-Displacement Housing Preference (ADHP)',
-        'aliceGriffith': 'Alice Griffith Housing Development Resident',
-        'rightToReturnSunnydale': 'Right to Return - Sunnydale'
-      }
+      preferenceMap =
```

This is unrelated, was just fixing a formerly silently failing test